src/main/java/nl/techiteasy/Techiteasy_part2/Models/Television.java

```diff
@@ -25,10 +26,20 @@ public class Television {
     public boolean ambilight;
     public int originalStock;
     public int sold;
+    @OneToOne
```

Comment:
Heel goed, television is de owner van deze relatie, dus in de television table staat een kolom voor remotcontrollerId